### PR TITLE
Set data verifier for Data Collector upon creation

### PR DESCRIPTION
**Pull request by cathinkaw**
Fixes the backend part of #644.

Because of the possible merge into volunteer reporting/react rewrite/proxy changes we haven't fixed the following:
- Listing all data verfiers in a dropdown list that you can choose from
- Set logged in user as the default data verifier
- Add this in the edit view

### CODE REVIEW

File: Source/UserManagement/Domain/DataCollector/DataVerifier/ChangeDataVerifierValidator.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace Domain.DataCollector.DataVerifier
+{
+    public class ChangeDataVerifierValidator : CommandInputValidatorFor<ChangeDataVerifier>
+    {
+        public ChangeDataVerifierValidator()
+        {
+            RuleFor(_ => _.DataCollectorId)
+                .NotEmpty().WithMessage("Data Collector Id must be set")
+                .SetValidator(new DataCollectorIdValidator());
+            //RuleFor(_ => _.DataVerifierId)
```
Comment (cathinkaw) [inline Source/UserManagement/Domain/DataCollector/DataVerifier/ChangeDataVerifierValidator.cs]: Outcommented as the serialization of the DataVerifierId (concept as) didn't work. Kept the code here as we should use a DataVerifierId and not a Guid when it works

Comment (einari) [reply Source/UserManagement/Domain/DataCollector/DataVerifier/ChangeDataVerifierValidator.cs]: Could you add an Issue describing the issue [here](https://github.com/dolittle-extensions/ReadModels.MongoDB/issues) with a reference to this file and how we can reproduce it ??

Comment (cathinkaw) [reply Source/UserManagement/Domain/DataCollector/DataVerifier/ChangeDataVerifierValidator.cs]: @woksin can you make the issue? You know more about the details than I.

**Comment by cathinkaw** [comment]
Well spotted @esp0. I have closed it as it is no longer relevant.

**Comment by einari** [comment]
This PR will be part of the #832 PR I have going. We merged it all into that as there was need to clean up user management and make it actually run before we merge it in with volunteer reporting into the new Reporting bounded context

**Pull request merged**